### PR TITLE
docs: add a note on byob

### LINKED ISSUE #59: request: provide an example of connecting to an existing browser
The documentation only shows how to connect to a cloud-hosted browser service: https://astral.deno.dev/advanced/connect/

Using Astral, how would you connect to a locally-running browser?

### PR DESCRIPTION
**Pull request by pixeleet**
fixes #59

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -137,3 +137,33 @@ const browser = await launch();
...
 ```
+
+## BYOB - Bring Your Own Browser
+
+
+Essentially the process is as simple as running a chromium-like binary with the following flags:
+
+```
+chromium --remote-debugging-port=<SOME_PORT> \
+--headless=new \
```
Comment (lino-levan) [inline README.md:148]: Just noting that `--headless=new` will change in the future, but this is fine for now.

File: README.md [hunk 2]
```diff
@@ -137,3 +137,33 @@ const browser = await launch();
...
+--no-first-run \
+--password-store=basic \
+--use-mock-keychain \
+--hide-scrollbars
+```
+
+> Technically, only the first one is necessary, though I've found that these flags generally get the best result. One this is running, it'd be as simple as
```
Comment (lino-levan) [inline README.md:155]:
```suggestion
Technically, only the first flag is necessary, though I've found that these flags generally get the best result. Once your browser process is running, connecting to it is as simple as
```

File: README.md [hunk 3]
```diff
@@ -137,3 +137,33 @@ const browser = await launch();
...
+
+```ts
+// Connect to remote endpoint
+const browser = await launch({
+  wsEndpoint: "wss://localhost:<SOME_PORT>",
```
Comment (lino-levan) [inline README.md:160]: Is wss correct here? I'd be surprised.

**Review by lino-levan — CHANGES_REQUESTED** [review]

File: README.md (hunk 3 above)
Comment (pixeleet) [reply README.md]: mm good point, tried an actual run and the launch seems to break on manual WS connection, see examples/byob.ts

File: README.md [hunk 4]
```diff
@@ -137,3 +137,23 @@ const browser = await launch();
...
+--password-store=basic \
+--use-mock-keychain \
+--hide-scrollbars
+```
+
+Technically, only the first flag is necessary, though I've found that these
+flags generally get the best result. Once your browser process is running,
+connecting to it is as simple as
+
+[Example](/examples/byob.ts) included in the `examples` directory.
```
Comment (lino-levan) [inline README.md:159]: Can we move this out of the examples directory? It's a little annoying to view this on the jsr so I'd prefer to just keep it inline.

File: README.md (hunk 1 above)
Comment (lino-levan) [reply README.md:148]: Probably fine to say nothing, just a note for someone in the future if this stops working.

File: examples/byob.ts [hunk 5]
```diff
@@ -0,0 +1,19 @@
+// Import Astral
+import { launch } from "jsr:@astral/astral";
+
+// Connect to remote endpoint
+const browser = await launch({
```
[the remark this answers is not in the document]
Comment (lino-levan) [reply examples/byob.ts:5]: Ah this is correct. I think I forgot this in my original comment, though the architecture was a little different in April so it's possible that it worked then. I'd be fine just saying `<WS-ENDPOINT-FROM-CONSOLE>` or something for now and revising an API later.

**Comment by pixeleet** [comment]
let me know if there's any more changes you'd like to see.

File: README.md [hunk 6]
```diff
@@ -137,3 +137,43 @@ const browser = await launch();
...
+
+```typescript
+// Import Astral
+import { launch } from "jsr:@astral/astral";
+
+// Connect to remote endpoint
+const browser = await launch({
+  wsEndpoint: "<WS-ENDPOINT>",
+  // args: ["--no-sandbox"],
+  // product: "chrome",
```
Comment (lino-levan) [inline README.md:167]: Could these be removed?

File: deno.jsonc [hunk 7]
```diff
@@ -15,6 +15,7 @@
   },
   "lock": false,
   "imports": {
+    "jsr:@astral/astral": "./mod.ts",
```
Comment (lino-levan) [inline deno.jsonc:18]: Unnecessary now

**Review by lino-levan — CHANGES_REQUESTED** [review]

**Comment by lino-levan** [comment]
Should be good after this

**Review by lino-levan — APPROVED** [review]

**Pull request merged**